addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/DimmerHandler.java

```diff
@@ -88,6 +93,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 output(ACTION_ZONELEVEL, 100, this.config.getFadeInTime());
             } else if (command.equals(OnOffType.OFF)) {
                 output(ACTION_ZONELEVEL, 0, this.config.getFadeOutTime());
+
+            } else if (command.equals(IncreaseDecreaseType.INCREASE)) {
+                output(ACTION_START_RAISING);
+            } else if (command.equals(IncreaseDecreaseType.DECREASE)) {
+                output(ACTION_START_LOWERING);
+            } else if (command.equals(StopMoveType.STOP)) {
+                output(ACTION_STOP_RAISING_LOWERING);
```

Comment:
I hadn't thought of querying since, at least with my Smart Bridge PRO, the level is reported at the end of the fade. I can't confirm how the "professional" bridges respond. 

On thinking through it this morning, I'm hesitant to add a query after the "stop" command as the Lutron Integration Guide says to allow 1500 ms for a query to execute. With a bank of four dimmers being "manually" faded, it seems as though there could be around a five-second delay between when the first is commanded to stop, and then the fourth is. (Yes, the response times I see below are better than the Lutron design specs.)

```
2017-03-17 07:12:38.131 [ItemCommandEvent          ] - Item 'pico_Test_Pico_down' received command ON
2017-03-17 07:12:38.137 [ItemStateChangedEvent     ] - pico_Test_Pico_down changed from OFF to ON
2017-03-17 07:12:38.139 [GroupItemStateChangedEvent] - group_pico_Test_Pico changed from OFF to UNDEF through pico_Test_Pico_down
2017-03-17 07:12:38.168 [ItemCommandEvent          ] - Item 'caseta_dimmer_Jeff_Office' received command DECREASE
2017-03-17 07:12:38.729 [ItemCommandEvent          ] - Item 'pico_Test_Pico_down' received command OFF
2017-03-17 07:12:38.735 [ItemStateChangedEvent     ] - pico_Test_Pico_down changed from ON to OFF
2017-03-17 07:12:38.740 [GroupItemStateChangedEvent] - group_pico_Test_Pico changed from UNDEF to OFF through pico_Test_Pico_down
2017-03-17 07:12:38.770 [ItemCommandEvent          ] - Item 'caseta_dimmer_Jeff_Office' received command STOP
2017-03-17 07:12:38.814 [ItemStateChangedEvent     ] - caseta_dimmer_Jeff_Office changed from 100.00 to 84.96
2017-03-17 07:12:38.820 [GroupItemStateChangedEvent] - group_room_Jeff_Office changed from 100.00 to 84.96 through caseta_dimmer_Jeff_Office
```
Trying to test this with a dimmer group reveals deeper problems though to be investigated before releasing this mainstream
```
2017-03-17 07:24:53.828 [ItemCommandEvent          ] - Item 'pico_Test_Pico_up' received command ON
2017-03-17 07:24:53.832 [GroupItemStateChangedEvent] - group_pico_Test_Pico changed from OFF to UNDEF through pico_Test_Pico_up
2017-03-17 07:24:53.836 [ItemStateChangedEvent     ] - pico_Test_Pico_up changed from OFF to ON
2017-03-17 07:24:53.856 [ItemCommandEvent          ] - Item 'group_room_Kitchen' received command INCREASE
2017-03-17 07:24:53.860 [ItemCommandEvent          ] - Item 'caseta_dimmer_Kitchen_Undercounter_Sink' received command INCREASE
2017-03-17 07:24:53.866 [ItemCommandEvent          ] - Item 'caseta_dimmer_Kitchen_Pendants' received command INCREASE
2017-03-17 07:24:53.870 [ItemCommandEvent          ] - Item 'caseta_dimmer_Kitchen_Cans' received command INCREASE
2017-03-17 07:24:53.875 [ItemCommandEvent          ] - Item 'caseta_dimmer_Kitchen_Undercounter_Range' received command INCREASE
2017-03-17 07:24:53.958 [ItemCommandEvent          ] - Item 'pico_Test_Pico_up' received command OFF
2017-03-17 07:24:53.960 [GroupItemStateChangedEvent] - group_pico_Test_Pico changed from UNDEF to OFF through pico_Test_Pico_up
2017-03-17 07:24:53.966 [ItemStateChangedEvent     ] - pico_Test_Pico_up changed from ON to OFF
2017-03-17 07:24:53.978 [WARN ] [lipse.smarthome.core.items.GroupItem] - Command 'STOP' has been ignored for group 'group_room_Kitchen' as it is not accepted.
2017-03-17 07:24:53.981 [ItemCommandEvent          ] - Item 'group_room_Kitchen' received command STOP
```
The good news is that the four dimmers get the INCREASE command logged in the span of ~20 ms. The bad news is that the STOP command isn't being communicated to the group members (all Lutron dimmers).